packages/desktop-client/src/components/accounts/TransactionsTable.js

```diff
@@ -1108,7 +1108,7 @@ function makeTemporaryTransactions(currentAccountId, lastDate) {
       date: lastDate || currentDay(),
       account: currentAccountId || null,
       cleared: false,
-      amount: 0,
```

Comment:
`0` is a value that the customer can actually provide.

`null` should be the default state for new transactions - i.e. transactions with no customer input